apps/rush-lib/src/data/Changelog.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Interface respresenting a changelog json object for a package used to represent the parsed
+ * content of CHANGELOG.json
+ *
+ * @alpha
+ */
+export interface IChangelog {
+  /**
+   * Name of the project
+   */
+  name: string;
+
+  /**
+   * Entries within the changelog corresponding to each published version.
+   */
+  entries: IChangeLogEntry[];
+}
+
+/**
+ * Interface representing a single published entry in the changelog.
+ *
+ * @alpha
+ */
+export interface IChangeLogEntry {
+  /**
+   * Published version for the entry. (Example: 1.0.0)
+   */
+  version: string;
+
+  /**
+   * Git tag used to identify the published commit. (Example: b7f55611e54910327a206476b185265498c66acf)
+   */
+  tag: string;
+
+  /**
+   * The UTC date when the publish was applied. (Example: Fri, 02 Dec 2016 22:27:16 GMT)
+   */
+  date: string;
```

Comment:
>date: string; [](start = 2, length = 13)

We should use Date.

Currently the change log processor has a bug where different date strings in the JSON file are passed through verbatim to the Markdown file.

The Markdown renderer should determine the date format, not the JSON file.